lib/questions.rb

```diff
@@ -1,22 +1,27 @@
 # keep only the elements that start with an a
 def select_elements_starting_with_a(array)
+  array.select{|element| element[0] == 'a'}
 end
 
 # keep only the elements that start with a vowel
 def select_elements_starting_with_vowel(array)
+  array.select{|element| element[0] =~ (/[aeiou]/)}
 end
 
 # remove instances of nil (but NOT false) from an array
 def remove_nils_from_array(array)
+  array.select{|element| element != nil}
 end
 
 # remove instances of nil AND false from an array
 def remove_nils_and_false_from_array(array)
+  array.select{|element| (element != nil &&  element != false) }
```

Comment:
Unnecessary spacing detected.<br>Prefer `!expression.nil?` over `expression != nil`.<br>Space missing to the left of {.<br>Space between { and | missing.